deploy/rbac-permissions-operator-config/01-osd-project-request.Template.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  annotations:
+    description: "Switches the ClusterROle for the 'admin' RoleBinding from 'admin' to 'dedicated-admins-project'."
+  name: osd-project-request
+  namespace: openshift-config
+objects:
+- apiVersion: project.openshift.io/v1
+  kind: Project
+  metadata:
+    annotations:
+      openshift.io/description: ${PROJECT_DESCRIPTION}
+      openshift.io/display-name: ${PROJECT_DISPLAYNAME}
+      openshift.io/requester: ${PROJECT_REQUESTING_USER}
+    creationTimestamp: null
+    labels:
+      name: ${PROJECT_NAME}
+    name: ${PROJECT_NAME}
+  spec: {}
+  status: {}
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: RoleBinding
+  metadata:
+    creationTimestamp: null
```

Comment:
wouldn't that just be created when applied?

Reply:
It was part of the generated template.. left it because of that else I'm OK to remove.  It works as it is though.

Reply:
i don't mind, was just wondering